packages/plugins/contracts/test/script/issues/Owned.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract Owned {
+    address public owner;
+
+    constructor(address _owner) {
+        owner = _owner;
+    }
+}
```

Comment:
thoughts reusing an existing contract in `MyContracts.sol`? imo it's better if we reuse contracts when possible to avoid having a bunch of contracts like this that are almost the same